### DISCUSSION TITLE
希望增加点击合集中其他视频时跳转至原始视频地址的功能

### DISCUSSION THREAD

**Discussion by fengsha990817** [the1812/Bilibili-Evolved#3440] [功能建议 / Ideas]
##### 这是直接点击切换的情况
![1 0](https://user-images.githubusercontent.com/38030455/174429966-4008a966-dc14-4f4e-a34a-2f7bb4af6d89.png)
![2 0](https://user-images.githubusercontent.com/38030455/174429964-ad6fc9fc-69a7-44ff-b349-a48576ca7156.png)
##### 这是通过空间中的视频链接进入的情况
![3 0](https://user-images.githubusercontent.com/38030455/174430088-61d144d7-a471-475b-84fb-18594206ecb3.png)

如上图所示，在切换合集中的不同视频时，地址栏的地址并不会进行切换，希望做到类似于稍后再看中跳转至原始视频地址的功能

**Comment by the1812** [comment]
这是 `av号转换` 的 bug (#3402)

**Reply by Geeyun-JY3** [reply]
现在被放进合集里的视频仍然有这个 bug，而且准确地说是 B 站官方的 bug：在无痕窗口打开 URL 中是 av 号的被放进合集里的视频，切换至所在合集内的任意视频，浏览器地址栏 URL 中的 av 号都不会变（跟踪参数倒是有可能会变）。

是否考虑针对这个官方 bug 进行优化？

**Reply by the1812** [reply]
官方 bug 我觉得应该跟官方反馈下？

**Reply by Geeyun-JY3** [reply]
主要是感觉官方在全面转向 BV 号后都不会怎么理 av 号，保证 av 号基本能用就不会管了，对官方我不抱太大希望🤦‍♂️像这个 bug 就至少放了 4 年（不会真的这 4 年都没人发现吧）……

**Reply by Geeyun-JY3** [reply]
> 在无痕窗口打开 URL 中是 av 号的被放进合集里的视频，切换至所在合集内的任意视频，浏览器地址栏 URL 中的 av 号都不会变（跟踪参数倒是有可能会变）。

现在还有官方程序员在看这个仓库？还是只是搞不知道什么东西的时候这 bug 顺便就被修了？
前两天就隐约感觉此 bug 疑似真的修了😧，今天实测真的没问题了，就拿我[上面几周前的评论](https://github.com/the1812/Bilibili-Evolved/discussions/3440#discussioncomment-18142877)说的情境为例：现在在无痕窗口打开 URL 中是 av 号的被放进合集里的视频，切换至所在合集内的任意视频，浏览器地址栏 URL 中的 av 号会正常变成目标视频的 BV 号了。
